README.md

```diff
@@ -37,7 +37,7 @@ Build the application bundles
 $ pnpm build
 
 # or
-$ pnpm build:bundles:watch
+$ pnpm build:watch
```

Comment:
unrelated to the rest of the changes, but `build:bundles:watch` did not exist in the `package.json`